partner_firstname/tests/test_user_form.py

```diff
@@ -48,17 +48,4 @@ def test_create_from_form_all(self):
         self.assertEqual(user_form.name, " ".join((firstname, lastname)))
 
     def get_clean_user_form(self):
```

Comment:
maybe this should be named `_get_user_form`, now that there is no cleaning

Reply:
done